lib/trln_argon/controller_override/local_filter.rb

```diff
@@ -18,6 +18,7 @@ def count_only
         builder.processor_chain.delete(:only_home_facets)
         builder.processor_chain.delete(:add_facet_paging_to_solr)
         builder.processor_chain.delete(:add_sorting_to_solr)
+        builder.processor_chain.delete(:add_document_ids_query)
```

Comment:
@mikedaul I think you don't want to delete this method from the processor chain here. By removing this method the local filter counts are wrong when a doc_ids query is present.

Since I rebased this branch on develop you'll have to `git reset --hard origin/TD-885-shareable-bookmarks` to pull down a fresh copy.